docs/reference/edot-collector/components/elasticsearchexporter.md

```diff
@@ -135,11 +135,43 @@ The exporter supports both internal batching and OpenTelemetry's standard `sendi
 
 ### Internal batching (default)
 
-By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue` and  `batcher` settings are defined.
+By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue` and/or  `batcher` settings are defined.
```

Comment:
also not technically correct. In previous versions, you can configure sending queue in front of an es exporter with internal buffering, i.e. double buffering.

Reply:
sending queue had nothing to do with batching, until `sending_queue::batch` becomes a config under `sending_queue`. But at a high level, queuing and batching are separate topics.

In the past, `batcher::enabled=false` disables both internal and exporterhelper batching, `batcher::enabled=true` disables internal batching but enables exporterhelper batching, `batcher::enabled=nil` enables internal batching but disables exporterhelper batching.

Reply:
ok so should we say


```suggestion
By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue::batch` and/or  `batcher` settings are defined.
```

Reply:
yes that's better